packages/default/scss/grid/_variables.scss

```diff
@@ -20,9 +20,9 @@ $grid-grouping-header-padding-y: $grid-grouping-header-padding-x !default;
 $grid-filter-cell-padding-x: $grid-padding-y !default;
 $grid-filter-cell-padding-y: $grid-filter-cell-padding-x !default;
 
-$grid-toolbar-padding-x: $toolbar-padding-x !default;
-$grid-toolbar-padding-y: $toolbar-padding-y !default;
-$grid-toolbar-spacing: $toolbar-spacing !default;
+$grid-toolbar-padding-x: $kendo-toolbar-padding-x-md !default;
```

Comment:
the grid-toolbar-padding and spacing vars are not being used anywhere and can be deleted.